### PR TITLE
Update zsh completion for 'docker service {create,update} {--endpoint…

### PR DESCRIPTION
**Pull request by sdurrheimer**
ref. #24581

@albers Thx for the ping

Signed-off-by: Steve Durrheimer s.durrheimer@gmail.com

### CODE REVIEW

**Comment by vdemeester** [comment]
LGTM 🐸
/cc @thaJeztah

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -1095,6 +1094,7 @@ __docker_service_subcommand() {
             _arguments $(__docker_arguments) \
                 $opts_help \
                 $opts_create_update \
+                "($help)--mode=[Limit Memory]:mode:(global replicated)" \
```
Comment (thaJeztah) [inline contrib/completion/zsh/_docker]: s/Limit Memory/Service Mode/ :smile:

Comment (sdurrheimer) [inline contrib/completion/zsh/_docker]: whoops fixed !

Comment (thaJeztah) [inline contrib/completion/zsh/_docker]: hehe, looks like no-one noticed last time :wink:

**Comment by thaJeztah** [comment]
LGTM, thanks!!

**Pull request merged**